dbt/adapters/bigquery/retry.py

```diff
@@ -113,16 +117,3 @@ def on_error(error: Exception):
                 raise FailedToConnectError(str(e))
 
     return on_error
-
-
-def _is_retryable(error: Exception) -> bool:
-    """Return true for errors that are unlikely to occur again if retried."""
-    if isinstance(
-        error, (BadGateway, BadRequest, ConnectionError, ConnectionResetError, ServerError)
```

Comment:
Only `BadRequest` is missing from BQ's defaults; the rest can be removed since they would be caught in `_job_should_retry`.

Reply:
We determined we can remove `BadRequest` here. The rationale is that we would expect a bad request to always be a bad request since the issue originates from the user side and not the BQ side.